libraries/abstractions/transport/utest/CMakeLists.txt

```
@@ -1,101 +1,89 @@
+            LWIP_COMPAT_SOCKETS=1
+            portHAS_STACK_OVERFLOW_CHECKING=1
+            portUSING_MPU_WRAPPERS=1
+            MPU_WRAPPERS_INCLUDED_FROM_API_FILE
+       )
 
 # ================= Create the library under test here (edit) ==================
 
-# The single source of truth for the transport_interface.h is in the coreMQTT repository.
-# In the FreeRTOS Device Software download page in AWS IoT Core, it is possible
-# to choose coreHTTP instead of coreMQTT. In this case, it is necessary to check
-# for the existence of the file in the downloaded software package.
-if( EXISTS ${AFR_MODULES_DIR}/coreMQTT/source )
-    set(transport_interface_dir "${AFR_MODULES_DIR}/coreMQTT/source/portable")
-elseif( EXISTS ${AFR_MODULES_DIR}/coreHTTP/source )
-    set(transport_interface_dir "${AFR_MODULES_DIR}/coreHTTP/source/portable")
-else()
-    message( FATAL_ERROR "No transport_interface.h exists for this included interface.")
-endif()
+# Include filepaths for source and include.
+include( ${AFR_ROOT_DIR}/libraries/abstractions/transport/transport_interface.cmake )
 
 # list the files you would like to test here
-    list(APPEND real_source_files
-                "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c"
-            )
+list(APPEND real_source_files
+            "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c"
+        )
 # list the directories the module under test includes
-    list(APPEND real_include_directories
-                .
-                "${transport_interface_dir}"
-                "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets"
-                "${AFR_ROOT_DIR}/libraries/abstractions/secure_sockets/include"
-                "${AFR_ROOT_DIR}/libraries/c_sdk/standard/common/include"
-            )
+list(APPEND real_include_directories
+            .
+            "${transport_interface_dir}"
+            "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets"
+            "${AFR_ROOT_DIR}/libraries/abstractions/secure_sockets/include"
+            "${AFR_ROOT_DIR}/tests/unit_test/linux/logging-stack"
+        )
 
 # =====================  Create UnitTest Code here (edit)  =====================
 
 # list the directories your test needs to include
-    list(APPEND test_include_directories
-                "${transport_interface_dir}"
-                "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets"
-                "${AFR_ROOT_DIR}/libraries/abstractions/secure_sockets/include"
-                "${AFR_ROOT_DIR}/libraries/c_sdk/standard/common/include"
-            )
+list(APPEND test_include_directories
+            "${transport_interface_dir}"
+            "${AFR_ROOT_DIR}/libraries/abstractions/transport/secure_sockets"
+            "${AFR_ROOT_DIR}/libraries/abstractions/secure_sockets/include"
+            "${AFR_ROOT_DIR}/tests/unit_test/linux/logging-stack"
```

I think that only this include path is required for the unit test file
```suggestion
            "${AFR_ROOT_DIR}/libraries/abstractions/secure_sockets/include"
```

---

transport_secure_sockets.h includes transport_interface.h,  iot_secure_sockets.h, "logging_levels.h"

transport_interface.h is in the path "${transport_interface_dir}"
iot_secure_sockets.h is in the path "${AFR_ROOT_DIR}/libraries/abstractions/secure_sockets/include"
logging_levels.h is in the path "${AFR_ROOT_DIR}/tests/unit_test/linux/logging-stack"

Therefore all these folder is needed resolving this comment.

Feel free to let me know if you suggest to make further changes, thank you.